### ISSUE TITLE
Question on the format of the input coverage file

### ISSUE THREAD

**Issue by kmin940** [liuyun313/IFCM#1]
In the HowToUseIFCM.doc, I cannot find the information on the format of the input coverage file.
Could you verify if the following TAB separated file format is appropriate? (Do I need a header, is it ok if it is tab separated?)

contig_id    sample_1    sample_2    sample_3  ...
contig-1     0.5              0.8               0.0            ...
contig-2     0.7              1.4               1.2            ...

Thank you very much.

**Comment by liuyun313** [comment]
IFCM is designed only based on k-mer features. The coverage value is the average coverage of a metagenome sequencing, not the coverage of contigs across different but related samples. Hope this massage is helpful to you. Do not hesitate to contact me if there is any problem.

**Comment by kmin940** [comment]
I see! Then is the following TSV file with a header appropriate? 
contig_id average_coverage
contig-1 0.7 ...
contig-2 1.1 ...
...
Thank you very much!

**Comment by liuyun313** [comment]
No. Only a numerical coverage value should be provided to estimate the species number. Coverage value is determined by a free software.
